frontend/src/app/main/ui/hooks/resize.cljs

```diff
@@ -89,21 +100,31 @@
                     start-size (mf/ref-val start-size-ref)
 
                     new-size (-> (+ start-size delta) (max min-val) (min max-val))]
-                (reset! current-size* new-size)
-                (swap! storage/user update-persistent-state file-id key new-size)))))
 
-         set-size
+                (set-size new-size)))))
+
+         on-resize-window
          (mf/use-fn
-          (mf/deps on-change-size file-id key)
-          (fn [new-size]
-            (let [new-size (mth/clamp new-size min-val max-val)]
-              (reset! current-size* new-size)
-              (swap! storage/user update-persistent-state file-id key new-size))))]
+          (fn []
+            (let [new-window-height (dom/get-window-height)]
+              (reset! window-height* new-window-height))))]
 
      (mf/with-effect [on-change-size current-size]
        (when on-change-size
          (on-change-size current-size)))
 
+     (mf/with-effect []
+       (.addEventListener glob/window "resize" on-resize-window)
+       (fn []
+         (.removeEventListener glob/window "resize" on-resize-window)))
+
+     (mf/with-effect [window-height]
+       (let [new-size (mth/clamp window-height min-val max-val)]
+         (set-size new-size)))
```

Comment:
This needs to work also when the resize axis is `:x` (ej in the inspect panel)

Reply:
This was a misunderstanding, I thought this was only supposed to work in vertical resizing areas